docs/source/gbd2017_models/risk_effects/smoking/index.rst

```diff
@@ -104,46 +106,96 @@ Vivarium Modeling Strategy
+  The mesh points for the relative risk curves for **former smokers** can be found here: /home/j/WORK/05_risk/risks/TEAM/sub_risks/tobacco/raw_data/metadata/rr/systematic_review_extraction_sheets/draws_for_PAF/426_lung_cancer/draws_quit.csv
 
-.. todo::
+The following code demonstrates how to assign relative risk values to individual simulants based on their exposure values.
 
-	Describe which entitity the relative risks apply to (incidence rate, prevalence, excess mortality rate, etc.) and *how* to apply them (e.g. :code:`affected_measure * (1 - PAF) * RR`). 
+.. code-block:: python
 
-  Be sure to specify the exact PAF that should be used in the above equation and either how to calculate it (see the `Population Attributable Fraction` section of the :ref:`Modeling Risk Factors <models_risk_factors>` document) or pull it (:code:`vivarium_inputs.interface.get_measure(risk_factor.{risk_name}, 'population_attributable_fraction')`, noting which affected entity and measure should be used)
+  from scipy.interpolate import interp1d
 
-.. todo::
+  """
+  rr_i =: simulant's individual relative risk
+  smoking_status_i =: simulant's smoking status exposure
+  draw_x =: selected draw for a given model run
+  sex_i =: simulant's sex
+  age_group_i =: simulant's age group
+  pack_year_exposure_i =: simulant's pack year exposure value, if applicable
+  years_since_quitting_exposure_i =: simulant's years since quitting exposure value, if applicable
+  """
 
-  Complete the following table to list the relative risks for each risk exposure category on the outcome. Note that if there are many exposure categories, another format may be preferable. 
+  if smoking_status_i == 'never':
+    rr_i = 1
 
-  Relative risks for a risk factor may be pulled from GBD at the draw-level using :code:`vivarium_inputs.interface.get_measure(risk_factor.{risk_name}, 'relative_risk')`. You can then calculate the mean value as well as 2.5th, and 97.5th percentiles across draws.
+  elif smoking_status_i == 'current':
 
-  The relative risks in the table below should be included for easy reference and should match the relative risks pulled from GBD using the above code. In this case, update the :code:`Note` below to include the appropriate :code:`{risk_name}`.
+    rr_current = pd.read_csv('/home/j/WORK/05_risk/risks/TEAM/sub_risks/tobacco/raw_data/metadata/rr/systematic_review_extraction_sheets/draws_for_PAF/426_lung_cancer/draws_pack.csv')
+    rr_current_i = rr_current.loc[rr_current.draw=draw_X].loc[rr_current.sex_id==sex_i].loc[rr_current.age_group_id==age_group_i]
+    x = rr_current_i.exposure.values
+    y = rr_current_i.rr.values
+    current_rr_function_i = interp1d(x, y)
 
-  If for any reason the modeling strategy uses non-GBD relative risks, update the :code:`Note` below to explain that the relative risks in the table are a custom, non-GBD data source and include a sampling strategy.
+    rr_i = current_rr_function_i(pack_year_exposure_i)
```

Comment:
Do you accept that the RR among former smoker quit recently can greater than current smoker at a random point? I am thinking about the extremes transit from current smoker to former smoker, or the opposite way.

Reply:
Yeah, according to this methodology, a current smoker with low pack years can increase their risk of lung cancer by quitting and becoming a former smoker with low years since quitting. We discussed this previously and it is a limitation of GBD methods, which I can add to this document.